src/lti_toolbox/models.py

```diff
@@ -89,7 +88,6 @@ class LTIPassport(models.Model):
     class Meta:
         """Options for the ``LTIPassport`` model."""
 
-        db_table = "lti_passport"
```

Comment:
as it is a django app, the table name is not prefixed by the app name ? it works only if `db_table` is not redefined ?

Reply:
exactly, here is the sql related to the migration I introduced:
```nc@xps15 ~/p/f/django-lti-toolbox (master)> DOCKER_USER=1000:1000 docker-compose run --rm django-lti-toolbox python sandbox/manage.py sqlmigrate lti_toolbox 0003_alter_lticonsumer_table_alter_ltipassport_table
Creating django-lti-toolbox_django-lti-toolbox_run ... done
🐳(entrypoint) creating user running in the container...
🐳(entrypoint) running your command: python sandbox/manage.py sqlmigrate lti_toolbox 0003_alter_lticonsumer_table_alter_ltipassport_table
System check identified some issues:

WARNINGS:
lti_toolbox.LTIPassport: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the LtiProviderAppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
BEGIN;
--
-- Rename table for lticonsumer to (default)
--
ALTER TABLE "lti_consumer" RENAME TO "lti_toolbox_lticonsumer";
--
-- Rename table for ltipassport to (default)
--
ALTER TABLE "lti_passport" RENAME TO "lti_toolbox_ltipassport";
COMMIT;
```

Reply:
This will be a pain in a blue/green context..
Maybe creating a view with the old name and then delete them is a solution ?

Reply:
This is already a pain out of blue/green context :
```  
[…]
  Applying core.0073_siteconfig_is_logo_enabled... OK
  Applying core.0074_video_transcode_pipeline... OK
  Applying deposit.0001_initial... OK
  Applying deposit.0002_noop_alter_filedepository_description_and_more... OK
  Applying deposit.0003_noop_alter_depositedfile_upload_state... OK
  Applying deposit.0004_alter_filedepository_playlist... OK
  Applying deposit.0005_alter_depositedfile_options... OK
  Applying lti_toolbox.0001_initial...Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 87, in _execute
    return self.cursor.execute(sql)
           ^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.DuplicateTable: relation "lti_passport" already exists
```